Javatrix.java

```diff
@@ -243,5 +243,26 @@ public static Javatrix constructWithCopy(double[][] arr)
         Javatrix actual = new Javatrix(arr);
         return actual;
     }
-}
 
+    /**
+     * Generate a matrix with random elements.
+     *
+     * @param m number of rows
+     * @param n number of columns
+     * @return a Javatrix object with matrix of random elements
+     */
+    public static Javatrix random(int m, int n)
+    {
+        Javatrix randomMatrix = new Javatrix(m, n);
+        for (int i = 0; i < m; i++)
+        {
+            for (int j = 0; j < n; j++)
+            {
+                double rand = java.lang.Math.random();
```

Comment:
Should we declare the rand variable outside the loop, then initialize its value inside, just so we're not creating a new variable for each iteration?